issue.go

```diff
@@ -23,40 +23,41 @@ type Issue struct {
 	UpdatedAt   time.Time `json:"updated_at,omitempty"`
 }
 
-func GetIssue(db *sql.DB, id int) (Issue, error) {
+func GetIssue(db *sql.DB, userId string) (Issue, error) {
```

Comment:
This function is wrong. GetIssue represent fetching one issue by it's id. You can't fetch one issue by userId. it does not make sense.

Reply:
yes i have to change just know identified that mistakee

Reply:
changed